brave/brave-core
# Fixed contribution queue not being deleted

Resolves https://github.com/brave/brave-browser/issues/7579

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
covered with unit tests

plan for QA
- enable rewards
- claim grant
- close browser
- copy db from corrupted profile (ask @NejcZdovc for it) into current profile
- start the browser
- make sure that no contribution is made
- open db and make sure that `contribution_queue` and `contribution_queue_publishers` is empty

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
